processor/attributesprocessor/attributes_metric_test.go

```diff
@@ -55,91 +54,19 @@ func generateMetricData(resourceName string, attrs map[string]interface{}) pmetr
 	res.Resource().Attributes().PutStr("name", resourceName)
 	sl := res.ScopeMetrics().AppendEmpty()
 	m := sl.Metrics().AppendEmpty()
-
-	switch m.Type() {
-	case pmetric.MetricTypeGauge:
-		dps := m.Gauge().DataPoints()
-		for i := 0; i < dps.Len(); i++ {
-			//nolint:errcheck
-			dps.At(i).Attributes().FromRaw(attrs)
-			dps.At(i).Attributes().Sort()
-		}
-	case pmetric.MetricTypeSum:
-		dps := m.Sum().DataPoints()
-		for i := 0; i < dps.Len(); i++ {
-			//nolint:errcheck
-			dps.At(i).Attributes().FromRaw(attrs)
-			dps.At(i).Attributes().Sort()
-		}
-	case pmetric.MetricTypeHistogram:
-		dps := m.Histogram().DataPoints()
-		for i := 0; i < dps.Len(); i++ {
-			//nolint:errcheck
-			dps.At(i).Attributes().FromRaw(attrs)
-			dps.At(i).Attributes().Sort()
-		}
-	case pmetric.MetricTypeExponentialHistogram:
-		dps := m.ExponentialHistogram().DataPoints()
-		for i := 0; i < dps.Len(); i++ {
-			//nolint:errcheck
-			dps.At(i).Attributes().FromRaw(attrs)
-			dps.At(i).Attributes().Sort()
-		}
-	case pmetric.MetricTypeSummary:
-		dps := m.Summary().DataPoints()
-		for i := 0; i < dps.Len(); i++ {
-			//nolint:errcheck
-			dps.At(i).Attributes().FromRaw(attrs)
-			dps.At(i).Attributes().Sort()
-		}
+	m.SetName("metric1")
+
+	// TODO: Add tests for other types.
```

Comment:
Can you link to an issue for doing this later?

Reply:
I actually don't think other types are necessary for this processor. So removed the comment